Resources/public/css/ezrecommendation.css

```diff
@@ -0,0 +1,6 @@
+.ezrecommendation {
+    padding: 20px;
+    -webkit-box-shadow: 4px 4px 50px -15px rgba(0,0,0,0.75);
+    -moz-box-shadow: 4px 4px 50px -15px rgba(0,0,0,0.75);
```

Comment:
Nitpick: you can drop support for prefixed properties here: `-webkit-box-shadow` and `-moz-box-shadow`. Please see: [box-shadow support](http://caniuse.com/#search=box-shadow)